src/components/calculator.js

```diff
@@ -0,0 +1,71 @@
+/* eslint react/prefer-stateless-function: [0] */
+import React from 'react';
+
+class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="container">
+        <div className="result">0</div>
+        <button type="button" className="operator-top">
+          AC
+        </button>
+        <button type="button" className="operator-top">
+          +/-
+        </button>
+        <button type="button" className="operator-top">
```

Comment:
- [ ] [OPTIONAL] You could consider creating a reusable `<Button>` component to make your project even better 👍

Reply:
@codepantha Any documentation or tutorial on how to do that? I'm new to react.